pom.xml

```diff
@@ -226,10 +226,10 @@
                             </execution>
                         </executions>
                     </plugin>
-
                     <plugin>
                         <groupId>org.kohsuke</groupId>
                         <artifactId>pgp-maven-plugin</artifactId>
+                        <version>1.1</version>
```

Comment:
I believe this is the addition needed to remove the warning.

Reply:
@Chanc3 what do you think of this change?

Reply:
Yes lets update that